POL-190 - Update FSM policies with new endpoint

chandravurubindi opened the pull request:
Update FSM policies with Credential service and Endpoint

### Description

Update FSM policies with new endpoint and Credential Service

### Issues Resolved

Update FSM policies with new endpoint and Credential Service
Update readme to replace credential instructions with instructions to use credential service

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### Running policy Link
SaaS Manager - Suspicious Users:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e8d85b2e8dac600012ec324

SaaS Manager - User Status Change:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e8d87231ccef500019d6033

SaaS Manager - Unsanctioned Spend:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e8d865f1ccef500019d6032

SaaS Manager - Renewal Reminder:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e8d840ee8dac600012ec320

Note:Tested all these policies with Credential Service and end point change (hard code Org ID value with 28010, since we don't have access to One Flexera Demo environment and other Org IDs:1105 and 27730 are not working).

cdwilhelm commented on saas/fsm/renewal_reminder/README.md:
```diff
@@ -19,11 +19,21 @@ This policy has the following input parameters required when launching the polic
 
-- Sends an email notification
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is  ...
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```
Looks like it was carried over from another policy

chandravurubindi replied:
Sorry,now its updated.

cdwilhelm commented on saas/fsm/suspicious_users/README.md:
```diff
@@ -18,11 +18,21 @@ This policy has the following input parameters required when launching the polic
 
-- Sends an email notification
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is  ...
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```
carried over from another policy

chandravurubindi replied:
updated now.

cdwilhelm commented on saas/fsm/unsanctioned_spend/README.md:
```diff
@@ -19,11 +19,21 @@ This policy has the following input parameters required when launching the polic
+## Prerequisites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is  ...
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```
same as the others.  carried over from another policy

chandravurubindi replied:
updated

cdwilhelm commented on saas/fsm/user_status_change/README.md:
```diff
@@ -19,11 +19,22 @@ This policy has the following input parameters required when launching the polic
 
-- Sends an email notification
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is  ...
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```
same as the others

chandravurubindi replied:
corrected.

chandravurubindi commented:
> Sorry, some more minor changes. I was reading the whole README file, not just your changes and saw that the Functional Details include obsolete details.
> 
> > Retrieve a Flexera IAM refresh token for authentication
> > The Flexera IAM refresh token must then be stored as a RightScale Credential in the account in which this policy will be applied. The credential must be named FSM_TOKEN.

@cdwilhelm - Thanks, I have removed the above details and added the below change in the Functional Details.
>>Please contact your Flexera Customer Success Manager for assistance to generate your FSM token.
Let me know for any other comments, Thanks.
